toqito/matrix_ops/perturb_vectors.py

```diff
@@ -0,0 +1,32 @@
+"""Perturb vectors is used to add a small random number to each element of a vector.
+
+A random value is added sampled from a normal distribution scaled by `eps`.
+"""
+
+import numpy as np
+
+
+def perturb_vectors(vectors: list[np.ndarray], eps: float = 0.1) -> list[np.ndarray]:
+    """Perturb the vectors by adding a small random number to each element.
+
+    :param vectors: List of vectors to perturb.
+    :param eps: Amount by which to perturb vectors.
+    :return: Resulting list of perturbed vectors by a factor of epsilon.
+
+    Example:
+        >>> from toqito.matrix_ops import perturb_vectors
+        >>> import numpy as np
+        >>> vectors = [np.array([1.0, 2.0]), np.array([3.0, 4.0])]
+        >>> perturbed_vectors = perturb_vectors(vectors, eps=0.1)
```

Comment:
```suggestion
        >>> perturb_vectors(vectors, eps=0.1)
```
Show the output as doctest verifies this.  You can use `# doctest: +SKIP` for doctest failures as we expect the output to be different due to `randn`.

https://github.com/vprusso/toqito/blob/4a4ce5e3e65b74348fcc1abe106b7ccf443bc050/toqito/rand/random_unitary.py#L24

Reply:
done!

Reply:
This is not done yet. See https://github.com/vprusso/toqito/pull/861#discussion_r1826984020